js/reference.js

```diff
@@ -551,11 +551,24 @@ var ERMrest = (function(module) {
                             notSet = false;
                             break;
                         }
-                    }
+                    } 
 
                     if (notSet) defaults.push(columnName);
                 });
-
+                
```

Comment:
The following lines of code are just removing any JSON columns from default list. This is incorrect. If it's `undefined` we should add the column to list of defaults. Also it's not doing what the comment suggests (changing "" to null).